src/main/java/org/jabref/logic/citationkeypattern/CitationKeyGenerator.java

```diff
@@ -105,7 +105,7 @@ public String generateKey(BibEntry entry) {
         String currentKey = entry.getCitationKey().orElse(null);
 
         String newKey = createCitationKeyFromPattern(entry);
-        newKey = replaceWithRegex(newKey);
+        newKey = newKey.replaceAll("\\\\[a-zA-Z]+\\{([^}]+)\\}", "$1");
```

Comment:
No. Please try to understand the comment https://github.com/JabRef/jabref/issues/12188#issuecomment-2471132857.